vscode-extension/media/CreateAIConfig.md

```diff
@@ -1,5 +1,11 @@
-## Create New AIConfig
+## Create AIConfig
 
-Open the Command Palette (CMD+Shift+P) and run the command `AIConfig: Create New`
+> _Tip_: You can use the Command Palette (`CMD`+`Shift`+`P`) to run `AIConfig: Create New` to create a new aiconfig anytime.
+
+You can create the aiconfig in JSON or YAML format. The file defines the models, prompts and settings
+
+> **Note**: Set the file extesnion to `*.aiconfig.json` or `*.aiconfig.yaml`. The extension will automatically open files with those extensions as a generative AI playground.
```

Comment:
typo: `extension`

Reply:
Great catch!

Reply:
"The extension will automatically open files with those extensions". Feels weird to use "extension" for two different definitions here. Ex: Let's keep "file extension" and say "the AIConfig Editor"

Reply:
Agreed :). Noticed that as well and fixed